src/java/relex/output/OpenCogSchemeLink.java

```diff
@@ -64,24 +65,50 @@ private String printLinks()
 	
 	private class LinkCB implements FeatureNodeCallback
 	{
+		String parseID;
 		String str;
 		public Boolean FNCallback(FeatureNode fn)
 		{
+			String lab = fn.get("LAB").getValue();
+			String lab_inst = fn.get("LAB").getValue() + "@" + parseID;
```

Comment:
You can't use the parse ID, because the sentence may have Ds**c in several places, and you won't know which is generated. You have to create a new uuid for each link.